src/main/webapp/app/entities/gene/gene.store.ts

```diff
@@ -2,22 +2,44 @@ import { IGene } from 'app/shared/model/gene.model';
 import { IRootStore } from 'app/stores';
 import axios from 'axios';
 import PaginationCrudStore from 'app/shared/util/pagination-crud-store';
-import { ICrudSearchAction } from 'app/shared/util/jhipster-types';
+import { ICrudGetAction, ICrudGetAllAction, ICrudSearchAction } from 'app/shared/util/jhipster-types';
+import { ensemblGeneClient, transcriptClient } from 'app/shared/api/clients';
+import { EnsemblGene, EnsemblGeneCriteria } from 'app/shared/api/generated';
+import { action, makeObservable, observable } from 'mobx';
+import { IEnsemblGene } from 'app/shared/model/ensembl-gene.model';
 
 const apiUrl = 'api/genes';
 const apiSearchUrl = 'api/_search/genes';
 
 export class GeneStore extends PaginationCrudStore<IGene> {
   searchEntities: ICrudSearchAction<IGene> = this.readHandler(this.getSearch);
+  ensemblGenes: EnsemblGene[] = [];
+  loadingEnsemblGenes = false;
```

Comment:
`loadingEnsemblGenes` seems to not be used. In `gene-details.tsx`, we have a state for the loading status.

Reply:
Good catch, this is legacy code. I will remove.